Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp

```diff
@@ -585,7 +597,7 @@ void Scene_points_with_normal_item::drawPoints(CGAL::Three::Viewer_interface* vi
 
     GLfloat point_size;
     viewer->glGetFloatv(GL_POINT_SIZE, &point_size);
-    viewer->glPointSize(4.f);
+    viewer->glPointSize(d->point_Slider->value());
```

Comment:
A note for later: I am pretty sure that the point size should be scaled according to the size of the bbox of the point set: let's say you have a point set contained in the bbox [0, 1]³, then the default point size (`5.`) will be huge for that point set.